Added github repo links and sample project on main page

0bserver07 opened the pull request:
In reply to issue #5 

There seems to be only 1 project so far with a public repository, or I couldn't find them with the Project title given on Projects page.

Later to be added.

Commit:
added repo link to project template + projects that have Repos

fchollet commented on build.py:
```diff
@@ -17,7 +17,7 @@
     f = open(fpath)
     metadata = {}
-    headers = {'title', 'date', 'category', 'contact', 'tagline', 'mailing_list'}
+    headers = {'title', 'date', 'category', 'contact', 'tagline', 'mailing_list','repository'}
```
Space after comma

fchollet reviewed (changes requested):
Please do not check anything into `website/`, that content is autogenerated. Added content should go in `templates/`, `'build.py`, etc.

Commit:
added Space after comma

Commit:
added Gitter Link to homepage + readme And Sample Project to homepage

Commit:
I fixed the /website part

0bserver07 commented:
1. fixed the /website (my bad first time using this py static site, usually Jekyll).
2. added the ai-on-example repo link.
3. added social-bot- repo link (people to find it)
4. added Gitter to main page and readme (I wasn't quite sure about the way I linked them though)

fchollet reviewed (changes requested):
Mention of the example project and the Gitter channel should go in /process. The landing page is for high-level info. Thanks!

Commit:
links for gitter + example moved to proces page. This closes #5, closes #7

0bserver07 commented:
All good now, wasn't sure if I should remove the gitter from readme.md or not though. (I thought it might help people when they find it on Github)

fchollet commented:
LGTM, thanks!
